src/pages/markets/components/sections/_other-markets.tsx

```diff
@@ -211,10 +211,10 @@ const Card = ({ market }: CardProps) => {
             <Text size="16px" weight="bold" mt="1.6rem">
                 {localize(markets_type[market].title)}
             </Text>
-            <Text size="16px" mt="0.8rem">
+            <Text size="15px" mt="0.8rem">
```

Comment:
Is this changed in the figma? may I know why are we changing the `font-size`?

Reply:
In figma I just checked it's 16px, I changed it because when I was zooming out below 60% the text was still out of bounds but as I have a second look at it, it works now even with 16px. So no need to change that.